app/api/v0/proofs/download/all/[block]/route.ts

```diff
@@ -34,23 +34,27 @@ export async function GET(
   const binaryBuffers: { binaryBuffer: Buffer; filename: string }[] = []
 
   for (const proofRow of proofRows) {
-    const team = await db.query.teams.findFirst({
-      columns: {
-        name: true,
-      },
-      where: (teams, { eq }) => eq(teams.id, proofRow.team_id),
-    })
+    const team = await getTeam(proofRow.team_id)
 
-    const { proof_type, cycle_type } = proofRow.cluster
     const teamName = team?.name ? team.name : proofRow.cluster_id.split("-")[0]
-    const filename = `block_${block}_${proof_type}_${cycle_type}_${teamName}.txt`
+    const filename = `${proofRow.block_number}_${teamName}_${proofRow.proof_id}.txt`
```

Comment:
Fine with this, but tiny aside: having a prefix here could help automatically sort any downloaded proofs together on the users machine... say, in the case it's downloaded into their general "Download" folder with other things.

Reply:
Ahh good call, I didn't notice that prefix. Will add it back

Reply:
Just realized that I did change the whole name. will add back what we had